client/packages/invoices/src/InboundShipment/DetailView/SidePanel/PricingSection.tsx

```diff
@@ -165,14 +166,16 @@ export const PricingSectionComponent = () => {
         </PanelRow>
         <PanelRow>
           <PanelLabel>{t('heading.rate')}</PanelLabel>
-          <PanelField>{currencyRate ?? 1}</PanelField>
+          <PanelField>{currencyRate === 0 ? 1 : currencyRate}</PanelField>
         </PanelRow>
         <PanelRow>
           <PanelLabel>{t('heading.total')}</PanelLabel>
           <PanelField>
-            {foreignCurrency(
-              pricing.foreignCurrencyTotalAfterTax ?? 0
-            ).format()}
+            {isHomeCurrency
+              ? '-'
```

Comment:
side thought: this has been used in a number of places and I wonder if we should create a global const for `UNDEFINED_STRING_VALUE` or similar?

Reply:
wasn't sure where to put it... so just created a file under common -> utils